workflows/parameters/CNRM-CM6-1-tasmin.yaml

```diff
@@ -23,6 +23,12 @@ jobs: |
       "variable_id": "tasmin",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "CNRM-CM6-1", "institution_id": "CNRM-CERFACS", "member_id": "r1i1p1f2", "grid_label": "gr", "version": "20180917" },
       "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp126", "table_id": "day", "variable_id": "tasmin", "source_id": "CNRM-CM6-1", "institution_id": "CNRM-CERFACS", "member_id": "r1i1p1f2", "grid_label": "gr", "version": "20190219" }
+<<<<<<< HEAD
```

Comment:
Heads up, @dgergel, there is a bit to clean up here. Looks like leftovers from a git merge conflict.